batch7crud-roll8/src/main/java/com/lftechnology/batch7crud/controller/EmployeeController.java

```diff
@@ -242,4 +244,45 @@ private int getCurrentPage(HttpServletRequest request) {
       return 1;
     }
   }
+
+  private String getAction(HttpServletRequest request) {
+    String[] parts = getPathParams(request);
+    String action;
+    if (parts.length == 0) {
+      action = AppConstants.FETCH;
+    }
+    else if (parts.length == 3 && AppConstants.CREATE.equals(parts[2])) {
+      action = AppConstants.CREATE;
+    }
+    else if (parts.length == 3) {
+      action = AppConstants.VIEW;
+    }
+    else if (parts.length == 4 && AppConstants.EDIT.equals(parts[3])) {
+      action = AppConstants.EDIT;
+    }
+    else {
+      action = null;
+    }
+    return action;
+  }
+
+  private String postAction(HttpServletRequest request) {
+    String[] parts = getPathParams(request);
+    String action;
+    System.out.println(request.getPathInfo());
+    System.out.println(parts.length);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)